### PR TITLE
Windows TS API Guardian

### PR DESCRIPTION
**Pull request by alan-agius4**
Check individual commits

### CODE REVIEW

**Comment by alan-agius4** [comment]
//cc @filipesilva

File: packages/animations/BUILD.bazel [hunk 1]
```diff
@@ -42,10 +42,10 @@ ng_package(
...
 ts_api_guardian_test_npm_package(
     name = "animations_api",
-    actualDir = "packages/animations/npm_package",
+    actualDir = "angular/packages/animations/npm_package",
     data = [
         ":npm_package",
         "//goldens:public-api",
     ],
-    goldenDir = "goldens/public-api/animations",
+    goldenDir = "angular/goldens/public-api/animations",
```
Comment (josephperrott) [inline packages/animations/BUILD.bazel]: By providing `angular/` at the beginning of the golden file, when bazel runs the `.accept` target, if the file does not already exist in the `goldens/` folder, it creates the `angular/goldens/` directory and places the golden there.

Comment (alan-agius4) [reply packages/animations/BUILD.bazel]: Umm that's a though one. Because it will not exist in the manifest :(

**Comment by filipesilva** [comment]
Nice work tracking down the path issue!

**Review by josephperrott — APPROVED** [review]
LGTM, I think we should get this in so that it works on windows as it currently blocks developers on windows from updating golden apis with the command locally.

As discussed offline, I will in another PR change ts-api-guardian to fail with better messaging if the `.accept` target is run without the golden file(s) already existing.

**Pull request closed without merging**